### PR TITLE
#20 Fix React warnings for unknown props in 15.2

### PR DESCRIPTION
**Pull request by danielmahadi**

### CODE REVIEW

File: src/Modal.jsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ class Modal extends React.Component {
 
     this._entering = this._entering.bind(this)
     this._exiting  = this._exiting.bind(this)
+    this._removeUnknownDivProps = this._removeUnknownDivProps.bind(this)
```
Comment (jquense) [inline src/Modal.jsx]: no need to bind the the method, it's not passed around and doesn't reference 'this'

File: src/Header.jsx [hunk 2]
```diff
@@ -18,13 +18,24 @@ class ModalHeader extends React.Component {
     onModalHide: React.PropTypes.func
   }
 
+  constructor() {
+    super();
+    this._removeUnknownDivProps = this._removeUnknownDivProps.bind(this)
```
Comment (jquense) [inline src/Header.jsx]: same here, no need to bind this method

**Comment by danielmahadi** [comment]
Hi @jquense ,

I have make the changes

**Comment by rudeayelo** [comment]
@danielmahadi could you please fix the conflicts and try to get this merged? 😉

**Comment by jquense** [comment]
I've actually fixed this already in a recent version I believe

**Pull request closed without merging**